modules/ROOT/pages/import-api-specification-design-center.adoc

```diff
@@ -0,0 +1,44 @@
+= Importing an API Specification from Design Center
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API Definition from Design Center into either into a new Mule Project, or an existing one.
+
+This task only works for RAML types of API specifications.
+
+== Importing an API from Design Center Into a New Project
+
+. In the top task bar, select *File*, then *New*, and then *Mule Project*.
+. Type a name for your project.
+. Select a Mule runtime engine version 4.1.4 or later.
+. In the *API Specification* section, select the *From Design Center or Local Files* tab.
+. In the *Location* field click the *...* button and select *Design Center...*.
+. Select your Anypoint Platform username. +
+If you are not logged in, you can click the *Add Account* button to add your Anypoint Platform account.
+. In the *Browse Design Center for APIs* window, select the Anypoint Platform business group where the API specification is located. +
+If you have configured your AnyPoint Platform credentials, your username and business group will be grayed out. +
+If you want to use a different account, click the *Add Account* button and log in again.
+. From the list of available APIs, select the one you want and click *OK*.
+. Click *Finish*.
+
+Studio scaffolds your API specification. If you choose not to create flows out of your API specification, uncheck the *Scaffold Flows From These API Specifications* option.
```

Comment:
This task topic is titled "Importing," so is it confusing to use "scaffolds" as the result of this task?
Also, at what point does the user unselect the Scaffold Flows checkbox? Should this step be in the previous task?

Reply:
If this option is really a checkbox...:
```suggestion
Studio scaffolds your API specification. If you choose not to create flows out of your API specification, unselect the *Scaffold Flows From These API Specifications* checkbox.
```